include/beman/any_view/detail/any_iterator.hpp

```diff
@@ -3,9 +3,9 @@
 #ifndef BEMAN_ANY_VIEW_DETAIL_ANY_ITERATOR_HPP
 #define BEMAN_ANY_VIEW_DETAIL_ANY_ITERATOR_HPP
 
-#include <beman/any_view/concepts.hpp>
-#include <beman/any_view/detail/intrusive_small_ptr.hpp>
-#include <beman/any_view/detail/iterator_adaptor.hpp>
+#include "../concepts.hpp"
```

Comment:
This sort of thing is a landmine if there exist more than one file named "concepts.hpp" anywhere in the universe. 
Anything in a header that is included by clients should use the same canonical name to avoid collisions or finding different files depending on spelling details. 
Headers by an implementation file might use a different spelling, however that does conflict with header module translations, and also risks the different file problem.

Reply:
@steve-downey Would you propose an auto-generated, Git-committed single-header include to be added to the project instead? I need this to work on Compiler Explorer, and when using system includes `<beman/any_view/...>` relative from `include`, it does not work.